### PR TITLE
Costing doc for stoichometric reactor 

### PR DESCRIPTION
**Pull request by avdudchenko**
## Fixes/Resolves:

This adds the costing documentation using new template format and removes costing description from model documentation. 

(replace this with the issue # fixed or resolved, if no issue exists then a brief statement of what this PR does)

## Changes proposed in this PR:
-
-

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: docs/technical_reference/costing/stoichiometric_reactor.rst [hunk 1]
```diff
@@ -9,45 +16,79 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
 .. csv-table::
    :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units"
 
-   "description", ":math:`Symbol_{example}`", "``parameter_name``", "1", ":math:`\text{dimensionless}`"
+   "Softening capital cost", ":math:`C_softening`", "``capital_cost_softening``", "374.9", ":math:`\text{USD/(lb/day)}`"
+   "Acid addition capital cost", ":math:`C_acid`", "``capital_cost_acid_addition``", "127.8", ":math:`\text{USD/(gallon/day)}`"
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/stoichiometric_reactor.rst]:
```suggestion
   "Softening capital cost", ":math:`C_{softening}`", "``capital_cost_softening``", "374.9", ":math:`\text{USD/(lb/day)}`"
   "Acid addition capital cost", ":math:`C_{acid}`", "``capital_cost_acid_addition``", "127.8", ":math:`\text{USD/(gallon/day)}`"
```

File: docs/technical_reference/costing/stoichiometric_reactor.rst [hunk 2]
```diff
@@ -9,45 +16,79 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
-   :header: "Description", "Symbol", "Variable Name", "Index", "Units"
-
-   "description", ":math:`Symbol_{example}`", "``variable_name``", "[t]", ":math:`\text{dimensionless}`"
+There are not unique costing variables constructed for stoichiometric reactor
 
 Capital Cost Calculations
 +++++++++++++++++++++++++
 
-Describe capital costs..keep it concise where possible
+(if user only includes reagents).(Refer to `stoichiometric reactor documentation <https://watertap.readthedocs.io/en/stable/technical_reference/unit_models/stoichiometric_reactor.html>`_)
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/stoichiometric_reactor.rst]:
```suggestion
If user only includes reagents, refer to `stoichiometric reactor documentation <https://watertap.readthedocs.io/en/stable/technical_reference/unit_models/stoichiometric_reactor.html>`_) 
```

File: docs/technical_reference/costing/stoichiometric_reactor.rst [hunk 3]
```diff
@@ -9,45 +16,79 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
-        C_{cap,tot} = C_{cap,example1}+C_{cap,example2}+C_{cap,other}
+        CC_{softening}=C_{softening}*\sum{M_{reagent}}
 
-    .. math::
+        CC_{acidification}=C_{acid}*\sum{Q_{reagent}}
 
-        C_{cap,example1} = fill in equation for each component in total capex equation
 
- 
+ where M_{reagent} is mass flow all reagent added in softening processes, and Q_{reagent} is flow volume of chemical added.
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/stoichiometric_reactor.rst]:
```suggestion
 where :math:`M_{reagent}` is mass flow all reagent added in softening processes, and :math:`Q_{reagent}` is flow volume of chemical added. 
```

File: docs/technical_reference/costing/stoichiometric_reactor.rst [hunk 4]
```diff
@@ -9,45 +16,79 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
 Costing Method Variables
 ++++++++++++++++++++++++
 
-The following variables are constructed on the unit block (e.g., m.fs.unit.costing) when applying the `cost_stoichiometric_reactor` costing method in the ``watertap_costing_package``:
-
-.. csv-table::
-   :header: "Description", "Symbol", "Variable Name", "Index", "Units"
-
-   "description", ":math:`Symbol_{example}`", "``variable_name``", "[t]", ":math:`\text{dimensionless}`"
+There are not unique costing variables constructed for stoichiometric reactor
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/stoichiometric_reactor.rst]:
```suggestion
There are not unique costing variables constructed for the stoichiometric reactor.
```

File: docs/technical_reference/costing/stoichiometric_reactor.rst [hunk 5]
```diff
@@ -1,6 +1,13 @@
 Stoichiometric Reactor Costing Method
 ======================================
 
+Currently, the costing method is implemented for lime and soda ash softening and acidification which only include
+the capital cost of building the reactor. The capital cost of lime soda ash is a function of 
+total reagent mass being added to the softening process and is only valid when both precipitant and reagents are provided.
+While acid addition capital cost is only constructed if only reagents are provided. Acid addition costing is 
+base on folume flow of acid per day. 
+(Please refer to to `stoichiometric reactor documentation <https://watertap.readthedocs.io/en/stable/technical_reference/unit_models/stoichiometric_reactor.html>`_ for details on dissolution and precipitation reaction configurations).
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/stoichiometric_reactor.rst]:
```suggestion
(Please refer to the `stoichiometric reactor documentation <https://watertap.readthedocs.io/en/stable/technical_reference/unit_models/stoichiometric_reactor.html>`_ for details on dissolution and precipitation reaction configurations). 
```

File: docs/technical_reference/costing/stoichiometric_reactor.rst [hunk 6]
```diff
@@ -1,6 +1,13 @@
 Stoichiometric Reactor Costing Method
 ======================================
 
+Currently, the costing method is implemented for lime and soda ash softening and acidification which only include
+the capital cost of building the reactor. The capital cost of lime soda ash is a function of 
+total reagent mass being added to the softening process and is only valid when both precipitant and reagents are provided.
+While acid addition capital cost is only constructed if only reagents are provided. Acid addition costing is 
+base on folume flow of acid per day.
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/stoichiometric_reactor.rst]:
```suggestion
While acid addition capital cost is only constructed if only reagents are provided. Acid addition costing is 
based on volume flow of acid per day. 
```

File: docs/technical_reference/costing/stoichiometric_reactor.rst [hunk 7]
```diff
@@ -9,45 +16,79 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
-
-    .. math::
-
-        C_{op,tot} = C_{op,example1}+C_{op,example2}+C_{op,other}
-
-    .. math::
-
-        C_{op,example1} = fill in equation for each component in total opex equation
-
+There are no operational costs provided, user needs to manually cost the reagent addition per mass/flow of reagent as in example below for
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/stoichiometric_reactor.rst]:
```suggestion
There are no operational costs provided, user needs to manually cost the reagent addition per mass/flow of reagent as done in the example below for 
```

File: docs/technical_reference/costing/stoichiometric_reactor.rst [hunk 8]
```diff
@@ -9,45 +16,79 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
-
-        C_{op,tot} = C_{op,example1}+C_{op,example2}+C_{op,other}
-
-    .. math::
-
-        C_{op,example1} = fill in equation for each component in total opex equation
-
+There are no operational costs provided, user needs to manually cost the reagent addition per mass/flow of reagent as in example below for 
+softening operation with soda ash and lime addition. 
+.. code-block::
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/stoichiometric_reactor.rst]:
```suggestion
softening operation with soda ash and lime addition. 

.. code-block::
```

**Review by MarcusHolly — COMMENTED** [review]
A couple of minor typos and formatting issues to address

File: docs/technical_reference/costing/stoichiometric_reactor.rst [hunk 9]
```diff
@@ -9,45 +16,80 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
-   :header: "Description", "Symbol", "Variable Name", "Index", "Units"
-
-   "description", ":math:`Symbol_{example}`", "``variable_name``", "[t]", ":math:`\text{dimensionless}`"
+There are not unique costing variables constructed for the stoichiometric reactor.
 
 Capital Cost Calculations
 +++++++++++++++++++++++++
 
-Describe capital costs..keep it concise where possible
+If user only includes reagents, refer to `stoichiometric reactor documentation <https://watertap.readthedocs.io/en/stable/technical_reference/unit_models/stoichiometric_reactor.html>`_)
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/stoichiometric_reactor.rst]:
```suggestion
If user only includes reagents, refer to `stoichiometric reactor documentation <https://watertap.readthedocs.io/en/stable/technical_reference/unit_models/stoichiometric_reactor.html>`_
```

**Review by MarcusHolly — APPROVED** [review]
LGTM - just one small typo I missed in a previous comment

File: docs/technical_reference/costing/stoichiometric_reactor.rst [hunk 10]
```diff
@@ -1,6 +1,13 @@
 Stoichiometric Reactor Costing Method
 ======================================
 
+Currently, the costing method is implemented for lime and soda ash softening and acidification which only include
+the capital cost of building the reactor. The capital cost of lime soda ash is a function of
```
Comment (OOAmusat) [inline docs/technical_reference/costing/stoichiometric_reactor.rst]: The capital cost of lime **and** soda ash is a function...

File: docs/technical_reference/costing/stoichiometric_reactor.rst [hunk 11]
```diff
@@ -1,6 +1,13 @@
 Stoichiometric Reactor Costing Method
 ======================================
 
+Currently, the costing method is implemented for lime and soda ash softening and acidification which only include
+the capital cost of building the reactor. The capital cost of lime soda ash is a function of 
+total reagent mass being added to the softening process and is only valid when both precipitant and reagents are provided.
+While acid addition capital cost is only constructed if only reagents are provided. Acid addition costing is
```
Comment (OOAmusat) [inline docs/technical_reference/costing/stoichiometric_reactor.rst]: Acid addition capital cost **is constructed only if** reagents are provided.

**Review by OOAmusat — APPROVED** [review]
@avdudchenko  LGTM - only a few minor edits required (see edits).

File: docs/technical_reference/costing/stoichiometric_reactor.rst (hunk 11 above)
Comment (avdudchenko) [reply docs/technical_reference/costing/stoichiometric_reactor.rst]: Done

**Pull request merged**